test/testinput/varobswriter_ukvnamelist_seviriasr.yaml

```diff
@@ -0,0 +1,106 @@
+time window:
+  begin: 2018-01-01T00:00:00Z
+  end: 2018-01-01T02:00:00Z
+
+observations:
+# This ob space shows what happens if the following are left as defaults
+# reject_obs_with_all_variables_failing_qc: false
+# reject_obs_with_any_variables_failing_qc: false
+# This means that despite the blacklist the entries still end up in the varobs file
+  - obs space:
+      name: SEVIRIASR
+      obsdatain:
+        engine:
+          type: H5File
+          obsfile: Data/varobs_ukvnamelist_seviriasr.nc4
+      simulated variables: [brightnessTemperature]
+      channels: 1, 3
+    obs filters:
+    # Set the flag of observations with missing values to "pass": we want to check if these
+    # values are encoded correctly in the VarObsFile.
+    - filter: Reset Flags to Pass
+      flags_to_reset: [10, 15]  # missing, Hfailed
+    # Reject observation 3: the blacklist doesn't stop the ob getting into the varobs because of the above "bad" settings
+    - filter: BlackList
+      where:
+      - variable:
+          name: MetaData/latitude
+        is_not_defined:
+    - filter: VarObs Writer
+      namelist_directory: ../etc/ukv/varobs
+      general_mode: debug
+    - filter: VarObs Checker
+      expected_main_table_columns:
+        # Only channels 1 and 3 are simulated and hence written to the VarObs file.
+        # In the arrays below, rows denote locations and columns are the varobs channels are consecutive.
+        field:    ["10","10","11","18","19","21","28","31","34","54","55","55","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80","80",
```

Comment:
Do you know why so many of these are equal to 80? Are you sure the nc4 file has been created correctly?

Reply:
I've been working on these ctests as you know, and this has persisted. I think it might just be a quirk of this data, as the files I'm now using were generated using the python script.